app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryAdapter.kt

```diff
@@ -148,5 +148,30 @@ class StorySummaryAdapter(
         }
       }
     }
+
+    private fun createRecyclerViewAdapter(
+      storyId: String,
+      chapterList: MutableList<ChapterSummary>
```

Comment:
This can just be a List<> (prefer not passing around mutable lists since it makes harder to reason about the state of data at any given point since mutable types can be changed).

Reply:
done